config/Neuer Ordner/borgcron_settings

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Configure the main settings for borgcron
+
+BACKUP_NAME='good-backup'
+REPOSITORY='ssh://user@somewhere.example:22/./dir'
+BACKUP_DIRS="/home /etc /srv /var/log /var/mail /var/lib /var/spool /opt /root /usr/local" #specify path without spaces, please
+
+BASE_DIR='/home/borg-backup' # optional
+PASSPHRASE_FILE="$BASE_DIR/good-key" #specify the passphrase, if your repo is encrypted
+COMPRESSION="lz4" # lz4 | zlib,6 | lzma,9
+
+PRUNE_PARAMS="--keep-daily=14 --keep-weekly=8 --keep-monthly=6 --keep-yearly=0"
+# for web servers (only disaster recovery:) --keep-daily=7 --keep-weekly=4 --keep-monthly=1 --keep-yearly=0
+
+# optional settings
+ARCHIVE_NAME="{hostname}-$BACKUP_NAME-{now:%Y-%m-%dT%H:%M:%S}" # or %Y-%m-%d
+ADD_BACKUP_PARAMS="" # --one-file-system for backing up root file dir
+
+SLEEP_TIME="5m" # time, the script should wait until re-attempting the backup after a failed one
```

Comment:
Not optional either… (script breaks when not present AFAIK, or has at least an error or so…)